package.json

```diff
@@ -13,15 +13,10 @@
     "medusa"
   ],
   "scripts": {
-    "clean": "cross-env ./node_modules/.bin/rimraf dist",
     "build": "medusa build",
-    "watch": "cross-env tsc --watch",
-    "test": "cross-env jest",
     "seed": "cross-env medusa seed -f ./data/seed.json",
```

Comment:
Let's change it to `medusa exec ./dist/helpers/seed.js`

Reply:
Sure

Reply:
Do we always run seeders from the compiled output? Because, we now run from the source as well.

```
node -r ts-node/require ./src/helpers/seed
```

Reply:
We created a new command `exec` that dynamically imports the file passed as a parameter so it has to be compiled